### DISCUSSION TITLE
NASA STAC Catalogues

### DISCUSSION THREAD

**Discussion by RichardScottOZ** [gjoseph92/stackstac#29] [Ideas]
Hi All,

Has anyone had a go and these and wrangling into stackstac?

**Comment by gjoseph92** [comment]
I think @scottyhq and @rmg55 have been a little bit? Check out https://gist.github.com/scottyhq/c4a4e889b58a0a153dd5fb18bad9f3e8 and https://github.com/rmg55/CloudDAAC_Binders/blob/main/s3_v_http.ipynb

**Comment by RichardScottOZ** [comment]
Thanks Gabe - saw an older one of his that was giving me a zero result, but that is likely a catalogue.

Don't think I have seen those ones yet, so thanks for that.

**Comment by RichardScottOZ** [comment]
and the bbox format is a string in one, I might have been using a list, giving problems

bbox = [-53.0172669999999968,-9.5331669999999988,-48.4956669999999974,-3.1035670000000000']

bbox = '-53.0172669999999968,-9.5331669999999988,-48.4956669999999974,-3.1035670000000000'

**Comment by scottyhq** [comment]
The NASA CMR STAC endpoint is very promising, but it is also somewhat experimental. There are a number of open issues to keep an eye on here https://github.com/nasa/cmr-stac . The endpoint is currently updated every two weeks or so, so your mileage may vary.

**Reply by RichardScottOZ** [reply]
Thanks Scott

**Comment by RichardScottOZ** [comment]
Also presumably what is USA and not USA I imagine?

**Comment by RichardScottOZ** [comment]
Anyway, borrowed Scott's gist for a quick look :-

https://nbviewer.jupyter.org/github/RichardScottOZ/Pangeo-Experiments/blob/main/stackstac-NASA-Landsat.ipynb

Comparison to element84 numbers returned - which had a lot missing for the bit of Brazil I was looking at yesterday.

I went with datetime as an explicit string, too

**Comment by RichardScottOZ** [comment]
and they don't currently all work it would seem?

https://cmr.earthdata.nasa.gov/stac/

**Reply by RichardScottOZ** [reply]
an error seeming slightly odd ```python SSLError: HTTPSConnectionPool(host='cmr.earthdata.nasa.gov.s3.amazonaws.com', port=443): Max retries exceeded with url: /stac/LARC_ASDC/collections?limit=500 (Caused by SSLError(SSLCertVerificationError("hostname 'cmr.earthdata.nasa.gov.s3.amazonaws.com' doesn't match either of '*.s3.amazonaws.com', 's3.amazonaws.com'")))```

**Reply by scottyhq** [reply]
@RichardScottOZ I'm having a hard time following what you're trying to do. If you're able to consolidate the comments into a single one at the top, or flesh out the context and error you're getting in a github issue in https://github.com/nasa/cmr-stac, or https://github.com/sat-utils/sat-search we could troubleshoot there.

**Comment by RichardScottOZ** [comment]
Ok, will do. At the moment, I was more just having a look at the 'get a quick list of what is available' capability: re element84 missing lots of Brazil data.

**Comment by RichardScottOZ** [comment]
@scottyhq   https://github.com/nasa/cmr-stac/issues/152

https://github.com/RichardScottOZ/Pangeo-Experiments/blob/main/STAC-Catalogue-NASA-LPDAAC-ECS-AST.ipynb
